pyproject.toml

```
@@ -0,0 +1,36 @@
+[build-system]
+requires = ['setuptools']
+build-backend = 'setuptools.build_meta'
+
+[tool.setuptools]
+packages = ['botbot']
+
+[project]
+name = 'botbot'
+version = '0.5.4'
```

I think this PR is deserving of at least a patch version bump, if not a minor version bump due to the edge-case breaking changes.
```suggestion
version = '0.5.5'
```